activesupport/lib/active_support/test_case.rb

```diff
@@ -66,10 +66,21 @@ def test_order
     alias :assert_not_respond_to :refute_respond_to
     alias :assert_not_same :refute_same
 
-    # Reveals the intention that the block should not raise any exception.
-    #
+
+    # Assertion that the block should not raise an exception.
+
+    # Without arguments, passes if evaluated code in the yielded
+    # block raises no exception.
+
     #   assert_nothing_raised do
-    #     ...
+    #     perform_service(param: 'no_exception')
+    #   end
+
+    # With arguments, passes if evaluated code in the yielded block
+    # raises no specified exception.
+
+    #   assert_nothing_raised(Services::PerformService::MissingInputData) do
```

Comment:
We don't assert this. Actually we don't recommend passing arguments. We should just remove it.